week-2-practise/level-2/6-map.js

```diff
@@ -4,6 +4,10 @@
 //    - each item in `percentages` formatted by your function
 
 var percentages = [1, 23, 92, 18];
+function persentage(index) {
+  return index + "%";
+}
+var percentagesFormatted = percentages.map(persentage);
```

Comment:
Good. I would probably call the functions something other than `percentage` though. It's creating a string from a number and adding a "%" sign, so you could call it something like `formatAsPercentage`. Something that makes it a bit clearer that it does.